content/docs/1.5.0/references/networking.md

```diff
@@ -28,8 +28,9 @@ From | Port | Protocol
 To | Port | Protocol
 --- | --- | ---
 `Other Longhorn Manager` | 9500 | TCP
-`Instance Manager` |  8500; 10000-30000 | TCP
-`Backing Image Manager` |  8000; 8001 | TCP
+`Instance Manager` | 8500; 8501 | TCP
+`Backing Image Manager` | 8000; 8001 | TCP
+`Backing Image Data Source` | 8001; 8002 | TCP
```

Comment:
I think Backing Image Data Source server and sync servers' port are `8000` and  `8001`
ref:
https://github.com/longhorn/backing-image-manager/blob/32cc0a79d209d1f6c2bd8323061e82f80b60970b/app/cmd/data_source.go#L22-L29

Reply:
It seems that the Longhorn manager does not need to send requests to the sync server (port 8001) of Backing Image Data Source pods as well. The Longhorn managers always talk with data source servers then data source servers proxies the requests to the sync server.

Similar for the Backing Image Manager.

In brief, Longhorn manager egress should be:
......
`Backing Image Manager` | 8000 | TCP
`Backing Image Data Source` | 8001 | TCP